tailwind.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
       },
       colors: {
         sauced: {
-          orange: "hsla(19, 100%, 50%, 1)",
+          orange: "hsla(19, 100%, 50%, var(--tw-bg-opacity))",
```

Comment:
This was to allow me to set the opacity via `bg-opacity-50` but it unlocks this colour now to be able to set an opacity on it. Prior to this, it was always 100% opacity.